src/NServiceBus.Testing.Tests/packages.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="ApiApprover" version="3.0.1" targetFramework="net452" />
-  <package id="ApprovalTests" version="3.0.11" targetFramework="net452" />
+  <package id="ApprovalTests" version="3.0.10" targetFramework="net452" />
```

Comment:
@mikeminutillo yes, it "restores" the same diff viewer behavior as in the core which lets you select the merge target. 3.0.11 always selects "received.txt" as the merge target and won't let you change it.